### ISSUE TITLE
Bug: Cannot generate a token with 'read:tasks' scopes.

### ISSUE THREAD

**Issue by kairoaraujo** [vmware/repository-service-tuf-api#187]
### What happened?

When running the FT, it was shown that it is impossible to generate a token with the scope `'read:tasks'`.

It is visible that `'read:tasks'` is missing in the code:

https://github.com/vmware/repository-service-tuf-api/blob/7b57e1778ec1608cb90d68a1490264e5ba4b3549/repository_service_tuf_api/token.py#L84-L92

### What steps did you take?

a) You can see it in the functional tests: https://github.com/vmware/repository-service-tuf/actions/runs/3526391298/jobs/5914262984

b) Generate a Token using API
1. Login using admin
2. Generate a token `/api/v1/token/new` adding scope `read:tasks`

c) Generate a token using CLI
1. Login using rstuf
2. Generate a token with scope `read:tasks`

### What behavior did you expect?

Create token with `'read:tasks'` is a must to have.

### Relevant log output

```shell
FAILED tests/functional/tokens/test_generate.py::test_admin_uses_http_api_to_generate_a_token[['write:targets', 'read:settings', 'read:tasks']-380] - AssertionError: {'detail': [{'ctx': {'given': 'read:tasks', 'permitted': ['read:bootstrap', 'read:settings', 'read:targets', 'read:tok...tted: 'read:bootstrap', 'read:settings', 'read:targets', 'read:token', 'write:targets'", 'type': 'value_error.const'}]}

FAILED tests/functional/tokens/test_generate.py::test_admin_uses_rstuf_command_line_interface_to_generate_token[write:targets read:targets read:tasks-380] - AssertionError: (1, '\x1b[31m╭─\x1b[0m\x1b[31m Error \x1b[0m\x1b[31m──────────────────────────────────────────────────────────────────...  \x1b[31m│\x1b[0m
554
  \x1b[31m╰──────────────────────────────────────────────────────────────────────────────╯\x1b[0m
555
  ')
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MVrachev** [comment]
I can work on this one.

**Comment by kairoaraujo** [comment]
Closed by #193
- #193 193
